R/utils-xportr.R

```diff
@@ -302,6 +302,16 @@ xpt_validate <- function(data) {
       glue("{fmt_fmts(names(chk_formats))} must have a valid format.")
     )
   }
+
+  # 4.0 max length of Character variables <= 200 bytes
+  max_nchar <- data %>%
+    dplyr::summarise(across(where(is.character), ~ max(nchar(., type = "bytes"))))
+  nchar_gt_200 <- max_nchar[which(max_nchar > 200)]
+  err_cnd <- c(
+    err_cnd,
+    glue("Length of {names(nchar_gt_200)} must be 200 bytes or less.")
```

Comment:
I believe that some users might not be aware that 200 bytes correspond to 200 characters in ASCII. While the message is fine, I am considering the possibility of mentioning that 1 byte equals 1 character, or specifically, that 200 bytes correspond to 200 characters for ASCII characters. 
What do you think?

Reply:
Thanks @cpiraux !  I think adding "200 bytes correspond to 200 characters in ASCII" is a good idea, I wasn't aware of it until now, good to know :)